src/components/AdminContainer.js

```diff
@@ -140,103 +143,140 @@ class AdminContainer extends Component {
     return (
       <React.Fragment>
         {adminAddress ? (
-          <Panel>
-            {baseStyle}
-            <div className="flex">
-              <div className="w-50">
-                <h1 className="mt0">Admin</h1>
-              </div>
-              <div className="w-50">
-                <NetworkSelectorContainer />
-              </div>
-            </div>
-            <div className="flex bb pb3">
-              <div className="w-50">
-                <h3>
-                  Current account{" "}
-                  <div
-                    style={{ cursor: "pointer" }}
-                    className="dib click-to-refresh"
-                    onClick={this.refreshCurrentAddress}
-                    title="Try to grab current account"
-                    tabIndex={1}
-                  >
-                    <i className="fas fa-sync-alt" />
+          <React.Fragment>
+            <nav className="dt pa3 pl6-l pr6-l w-100 border-box ph5-ns bg-white shadow-1-ns items-center flex-l flex-row-l">
+              <a href="/" title="Home">
+                <img
+                  src={"../../static/images/logo.svg"}
+                  className="dtc v-mid mid-gray link dim pa3"
+                  style={{ width: "200px" }}
+                  alt="OpenCerts"
+                />
+              </a>
+              <div className="w-80-ns flex-l flex-row-l justify-end">
+                <div className="w-100-s pl2-l pr2-l">
+                  <h3 className="ma0">
+                    Current Account{" "}
+                    <div
+                      style={{ cursor: "pointer" }}
+                      className="dib click-to-refresh"
+                      onClick={this.refreshCurrentAddress}
+                      title="Try to grab current account"
```

Comment:
I dont understand this sentence, what's the goal of the action ?

Reply:
If the user changes to other wallet, they can manually refresh so that it detects the newly provided wallet.

Reply:
> The deploying toast is really weird
> 
> ![image](https://user-images.githubusercontent.com/503185/63664579-b3de8680-c7f9-11e9-931c-d6bb07581c71.png)
> 
> It doesnt wait for the deployment to be done which makes me feel like: "did it work ? what happened ?"
> 
> In the end what's the goal with toaster ? there is already a loading indicator on the deploy button ?

Removed.